### PR TITLE
Fix problem with constraints

### PR DESCRIPTION
**Pull request by vitali-voropaiev**
Removes constraints before adding new constraints on toggling a control.

### CODE REVIEW

File: AnimatedTextInput/Classes/AnimatedTextInput.swift [hunk 1]
```diff
@@ -93,6 +93,7 @@ public class AnimatedTextInput: UIControl {
     // MARK: Configuration
 
     private func addLineViewConstraints() {
+        self.removeConstraints(self.constraints)
         pinLeading(toLeadingOf: lineView, constant: style.leftMargin)
```
Comment (danielmartin) [inline AnimatedTextInput/Classes/AnimatedTextInput.swift]: Nit: For style consistency with the rest of the code, I'd remove`self` in this case.

**Review by danielmartin — APPROVED** [review]
LGTM

**Comment by victorBaro** [comment]
Good catch. However, I would move it to a new method `removeAllConstraints()` called from `updateConstraints()`. It is misleading to have it inside `addLineViewConstraints()`.

**Comment by victorBaro** [comment]
@Cronos could you fix the conflicts so we can merge?

**Pull request merged**